### ISSUE TITLE
PDF version

### ISSUE THREAD

**Issue by mrshu** [ericdrowell/BigOCheatSheet#27]
Since this is a cheatsheet I'd say it would be nice to also have it in a PDF form for offline/print use.

What do you think?

**Comment by aloisdg** [comment]
@ericdrowell Any update on this? I really like @crepererum 's idea. You could also have a "print" button on your page. During the time, @ro31337 made a [version](https://github.com/ro31337/bigoposter) without credits and strange [Open Source philosophy](https://github.com/ro31337/bigoposter#star-to-open-source).

**Comment by ericdrowell** [comment]
@mrshu @crepererum @josem @aloisdg completely agree, and had been planning on adding a print button or PDF button for awhile, but have been pretty busy.  It's clear that a lot of folks want this, so I'll try and set this up today.  It would have been nice if @ro31337 had tried to set something up in this repo, rather than trying to rip it off.  Oh well.

**Comment by josem** [comment]
Hi @ericdrowell! Thanks so much for the update! 

I'm sorry if you feel that I ripped it off, I didn't get any benefit for creating a new repo, I didn't add ads or anything and the only reason I did it was because this issue was open over 3 years ago and I thought that this project was not maintained anymore and I could add some improvements with a new approach. Again, I'm sorry if that was not the case.

If this project keeps evolving and bigoref.com doesn't make sense anymore I'll remove it and try to contribute to this one instead. You can also see that I mentioned you and this project more than once in the readme file: https://github.com/josem/bigoref giving you all the credit for the project.

Thanks so much for all the work on this!

**Comment by ericdrowell** [comment]
@josem no we were talking about @ro31337 https://github.com/ro31337/bigoposter :P

**Comment by josem** [comment]
Ah awesome, sorry! I thought you were talking about bigoref.com since you said it wouldn't be the first time, anyway, sorry if this hasn't felt great to you. Thanks so much for creating this! 👍 💯

**Comment by ericdrowell** [comment]
@josem no problem at all, I'm glad that people have gotten some use out of it.

**Comment by ericdrowell** [comment]
@mrshu @crepererum @josem @aloisdg all done.  Here's what I did:

* Added a new "Download PDF" button at the top.
* when you click this button, it opens up a new page with just the legend, tables, and a new chart I made today
* it uses pdfcrowd to convert the page into a PDF sized for 8.5'' wide printing paper and then automatically downloads it

P.S. I was talking to a friend of mine who is a super talented visualization designer, and we will more than likely also create a really cool poster version of the information
